hack/build/run-functional-tests.sh

```diff
@@ -38,8 +38,16 @@ arg_oc="${KUBECTL:+-oc-path=$KUBECTL}"
 test_args="${test_args} -ginkgo.v ${arg_master} ${arg_namespace} ${arg_kubeconfig} ${arg_kubectl} ${arg_oc}"
 
 echo 'Wait until all CDI Pods are ready'
-while [ -n "$(./cluster/kubectl.sh get pods -n $CDI_NAMESPACE -o'custom-columns=status:status.containerStatuses[*].ready' --no-headers | grep false)" ]; do
-    sleep 5
+retry_counter=0
+while [ $retry_counter -lt 30 ] && [ -n "$(./cluster/kubectl.sh get pods -n $CDI_NAMESPACE -o'custom-columns=status:status.containerStatuses[*].ready' --no-headers | grep false)" ]; do
+	retry_counter=$((retry_counter + 1))
+    sleep 10
 done
 
+if [ $retry_counter eq 30 ]; then
```

Comment:
might be nice if 30 is a readonly defined at the beginning of script so it's easier to find and change.